entries/n/ngpvan.com.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Site Name": {
+    "domain": "ngpvan.com",
+    "img": "ngpvan.com.png",
+    "tfa": ["sms", "totp"],
+    "url": "https://myaccount.ngpvan.com/Account",
+    "documentation": "https://www.ngpvan.com/actionid/",
```

Comment:
I don't think this documentation URL is required. There aren't any instructions after registration, plus the ActionID portal has it on the settings page clearly.

```suggestion
```